lib/consts.json

```diff
@@ -0,0 +1,29 @@
+{
+  "MISSING_OR_INCORRECT_CREDENTIALS_ERROR": "Missing or incorrect credentials - please select valid access and secret keys first",
+  "DEFAULT_CREDENTIAL_LABELS": ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "REGION"],
+  "MAX_AUTOCOMPLETE_RESULTS": 100,
+  "ALL_AWS_REGIONS": [
```

Comment:
This has been a long debate about this. Initially I've created the function to retrieve dynamically all of the regions from AWS library and you're completely right - it's not a big overhead and it's objectively the better solution. There is however a big number of problems with that – starting from the fact that every service that use the "basic" set of regions (meaning the EC2 ones) do not provide a method in the API to retrieve the regions. This means, that to access the region list from those plugins, we'd need to communicate with EC2 which would be fine, but requires the account used with any given AWS plugin to have permissions to access EC2. Not to mention the fact that EC2 does not provide user friendly names via API – this would require to be taken from Lightsail (as specified in AWS docs - https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) which require another set of accesses and does not provide the same amount of regions as EC2. These are all the reasons why Kenny, David and I decided to go with this static list. I do realise there is a potential maintenance involved with keeping this list up-to-date, but that's just seems like a better decision to update it once in a blue moon than to force users tinkering with S3 or Lambdas to grant permissions to the service(s) they do not intent to use to their service accounts.

Reply:
Sounds reasonable, I need to learn more about AWS then. I'll create a task to create a test verifying that the regions at AWS did not change.